src/view/Input.js

```diff
@@ -0,0 +1,18 @@
+import Console from '../utils/Console.js';
+import Message from '../constant/Message.js';
+
+const { INPUT } = Message;
+
+class Input {
+  static async carName() {
+    const input = await Console.readLineAsync(INPUT.car_name);
+    return input;
```

Comment:
아래처럼 작성하시는 방법도 있었을텐데 위와 같이 작성하신 이유가 있을까용?

```suggestion
    return await Console.readLineAsync(INPUT.car_name);
```

Reply:
변수에 저장하는게 좀 더 읽기 쉬운? 느낌이 들었던 것 같아요. 지금 생각해보니 굳이? 변수를 사용 했었어야 했나 싶은 부분이기도 하네요....감사합니다:)